eng/pipelines/templates/stages/apiview-python-ci.yml

```diff
@@ -0,0 +1,121 @@
+trigger:
+  branches:
+    include:
+      - main
+      - feature/*
+      - release/*
+      - hotfix/*
+  paths:
+    include:
+      - eng/pipelines/templates/stages/apiview-python-ci.yml
+      - packages/python-packages/api-stub-generator
+
+pr:
+  branches:
+    include:
+      - main
+      - feature/*
+      - release/*
+      - hotfix/*
+  paths:
+    include:
+      - eng/pipelines/templates/stages/apiview-python-ci.yml
+      - packages/python-packages/api-stub-generator
+
+variables:
+  PythonVersion: '3.10.2'
+  PythonParserPackagePath: 'packages/python-packages/api-stub-generator'
+  FeedName: 'public/azure-sdk-api-review-tools'
+  ArtifactName: 'apiviewparserpython'
+
+stages:
+  - stage: 'Build'
+    jobs:
+      - job: 'Build'
+
+        pool:
+          name: azsdk-pool-mms-ubuntu-2004-general
+          vmImage: MMSUbuntu20.04
+
+        steps:
+          - template: /eng/pipelines/templates/steps/use-python-version.yml
```

Comment:
When did we add this template to the tools repo? Is this something worth sharing with the python repo?

Reply:
https://github.com/Azure/azure-sdk-tools/issues/3064